cmd/client-s3.go

```diff
@@ -46,12 +46,22 @@ import (
 	"github.com/minio/minio-go/v7/pkg/policy"
 	"github.com/minio/minio-go/v7/pkg/replication"
 	"github.com/minio/minio-go/v7/pkg/sse"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
 
 	"github.com/minio/minio-go/v7/pkg/s3utils"
 	"github.com/minio/minio-go/v7/pkg/tags"
 	"github.com/minio/minio/pkg/mimedb"
 )
 
+var mcS3GETLatency = promauto.NewGaugeVec(
+	prometheus.GaugeOpts{
+		Name: "mc_s3_get_latency",
+		Help: "The time-to-first-byte duration of the last GET request",
```

Comment:
> Why is the last request interesting? Assuming this is just inaccurate text.

Because this is Prometheus, the value of TTFB will get scarped in each X seconds.. and users will monitor it

Reply:
@vadmeste ok, you mean "latest", not last as in the "final read".
```suggestion
		Help: "The duration of the latest GET request read",
```